### PR TITLE
mrc-226 add basic report page

### PR DESCRIPTION
**Pull request by hillalex**
Includes commits from https://github.com/vimc/orderly-web/pull/40 so merge that one first.

This PR adds the scaffolding for the single report page. 

In adding our first secured route (the report page is locked down to report readers) I discovered that we hadn't wired our authorisation generator in to the `IndirectClient`s! So have added that in here also.

### CODE REVIEW

File: src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/GithubWebTests.kt [hunk 1]
```diff
@@ -64,4 +69,15 @@ class GithubWebTests : SeleniumTest()
                 .contains("We have not been able to successfully identify you as a member of the app's configured Github org")
 
     }
+
+    @Test
```
Comment (EmmaLRussell) [inline src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/GithubWebTests.kt]: should there be a test that report readers can see the report page a well?

File: src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/GithubWebTests.kt [hunk 2]
```diff
@@ -64,4 +69,15 @@ class GithubWebTests : SeleniumTest()
...
+    fun `non report readers cannot see report page`()
```
Comment (EmmaLRussell) [inline src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/GithubWebTests.kt]: Should there be a test that report readers can see the page as well? Oh, I see you've got one for the Montagu login though.

File: src/app/templates/report.ftl [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" data-toggle="tab" href="#downloads" role="tab">Downloads</a>
+                </li>
+            </ul>
+            <hr/>
+        </div>
+        <div class="col-12 col-md-8 col-xl-9 tab-content">
+            <div class="tab-pane active" role="tabpanel" id="report">
+                <#include "partials/report.ftl">
```
Comment (EmmaLRussell) [inline src/app/templates/report.ftl]: Maybe confusing to the have the full template and partial file both called report.ftl? Could be report details?

File: src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/GithubWebTests.kt (hunk 2 above)
Comment (hillalex) [reply src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/GithubWebTests.kt]: Yeah this was just confirming that authorization was working for GitHub. I'll change the test name to make that clearer.

**Comment by hillalex** [comment]
Re the test user, yes, for some reason I've added that in a different branch but you're right, you need it here too. I'll add.

File: dev/cli.sh [hunk 4]
```diff
@@ -2,5 +2,5 @@
 
 git_id=$(git rev-parse --short=7 HEAD)
 
-image=docker.montagu.dide.ic.ac.uk:5000/orderly-web-cli:$git_id
+image=docker.montagu.dide.ic.ac.uk:5000/orderly-web-cli:fixcli
```
Comment (hillalex) [inline dev/cli.sh]: This cli is broken at the moment. Once this is merged: https://github.com/vimc/orderly-web/pull/41 can return this to $git_id

**Comment by hillalex** [comment]
Actually looks like the test user is being added, it's just the cli is broken. Have hard-coded the cli image to a working branch.

**Pull request merged**